libfwupdplugin/fu-device.c

```diff
@@ -1848,9 +1844,7 @@ fu_device_set_physical_id (FuDevice *self, const gchar *physical_id)
 const gchar *
 fu_device_get_physical_id (FuDevice *self)
 {
-	FuDevicePrivate *priv = GET_PRIVATE (self);
-	g_return_val_if_fail (FU_IS_DEVICE (self), NULL);
-	return priv->physical_id;
+	return fwupd_device_get_physical_id (FWUPD_DEVICE (self));
```

Comment:
I would recommend to instead change `fu-device.h` to map the function like other fu_* -> fwupd_* maps happen.